src/Trakx.CryptoCompare.ApiClient.Websocket/CryptoCompareWebsocketConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Trakx.Utils.Attributes;
+
+namespace Trakx.CryptoCompare.ApiClient.Websocket
+{
+    public class CryptoCompareWebsocketConfiguration
```

Comment:
let's make it an init only record and suppress the warnings or add a constructor for those props